modules/tensor_mechanics/test/tests/notched_plastic_block/tests

```diff
@@ -0,0 +1,85 @@
+[Tests]
+  [./cmc_planar]
```

Comment:
FYI - Andrew recently enhanced the MOOSE doc system to handle cases where we have similar requirements that vary only on a single parameter, or other condition. We create a sunblock surrounding the grouped tests and list the requirement in the top level subblock. Then in each individual test we use "detail = sentence fragment". These details all get assembled into a bulleted list for a single requirement in the docs.

See the example here (very bottom of the page):
https://mooseframework.inl.gov/python/MooseDocs/generate.html

Reply:
Done this, thanks @permcody